config/application.rb

```diff
@@ -16,6 +16,7 @@
 # require "action_cable/engine"
 # require "sprockets/railtie"
 require "rails/test_unit/railtie"
+require "govspeak"
```

Comment:
Looks quite odd to require this here why not just do so in  app/controllers/govspeak_test_controller.rb?